src/Entity/Pet.php

```diff
@@ -75,4 +91,35 @@ public function setAge(int $age): self
 
         return $this;
     }
+
+    /**
+     * @return Collection|Photo[]
+     */
+    public function getPhotos(): Collection
+    {
+        return $this->photos;
+    }
+
+    public function addPhoto(Photo $photo): self
+    {
+        if (!$this->photos->contains($photo)) {
+            $this->photos[] = $photo;
+            $photo->setPet($this);
+        }
+
+        return $this;
+    }
+
+    public function removePhoto(Photo $photo): self
+    {
+        if ($this->photos->contains($photo)) {
+            $this->photos->removeElement($photo);
+            // set the owning side to null (unless already changed)
+            if ($photo->getPet() === $this) {
```

Comment:
I don't really like `if` in a `if` alternative is to invert the first `if` and do:

```
if (!$this->photos->contains($photo)) {
    return $this;
}
```

and then do the rest.